hushline/model.py

```diff
@@ -54,57 +54,57 @@ def check_password(self, plaintext_password: str) -> bool:
 
     @property
     def totp_secret(self) -> str | None:
-        return decrypt_field(self._totp_secret)
+        return decrypt_field(self._totp_secret, domain=b"totp_secret")
```

Comment:
@micahflee @jeremywmoore These row encryptions aren't making use of the ``aad`` kwarg yet. I need more insight from the team to around which value(s) could be used as a unique / canonical representation of a user. Best if it doesn't ever change for the duration of the user's admittance to the database. If it does change, then all the encrypted values will have to be re-encrypted under the new context. Possible ideas: email address, user-id, public pgp key...

Reply:
I'd love for @glenn-sorrentino to weigh in on this one as it feels like a product question :)

Reply:
Addressed by (https://github.com/scidsg/hushline/pull/411/commits/8ca05bac11d28b417a275761e93b648130f135ea)